README.md

```diff
@@ -485,6 +485,15 @@ servers:
 | format    | `string` | Format of files, such as `parquet`, `delta`, `json`, `csv`                                                       |
 | delimiter | `string` | (Only for format = `json`), how multiple json documents are delimited within one file, e.g., `new_line`, `array` |
 
+#### AWS Kinesis Data Streams Server Object
+
+| Field  | Type     | Description                                                               |
+|--------|----------|---------------------------------------------------------------------------|
+| type   | `string` | `kinesis`                                                                 |
+| name   | `string` | The name of the Kinesis data stream.                                      |
+| region | `string` | AWS region, e.g., `eu-west-1`.                                            |
+| format | `string` | The format of the records. Examples: json, avro, protobuf.                |
```

Comment:
looking at AWS Glue example, let's add account as well:
`| account  | `string` | REQUIRED. The AWS account, e.g., `1234-5678-9012`          |`

Reply:
same here, account is not needed to connect to a kinesis stream. But yeah looking at all server objects it looks like they are not all following the same pattern, so maybe. Does it help with having the account id? I don't think so tbh

Reply:
Probably _not_ since when you would call this tool you are hopefully already in SSO environment and need only the name. Nice!